client/src/store.js

```diff
@@ -20,10 +20,31 @@ export default new Vuex.Store({
   },
   actions: {
     async fetchPipelines({ commit, state }) {
-      if (state.pipelines.length === 0) {
+      if (Object.keys(state.pipelines).length === 0) {
         const { data } = await getPipelineList();
-        commit('setPipelines', data);
-        return data;
+        // Sort list of pipelines in each category by name
+        Object.keys(data).forEach((key) => {
+          const category = data[key];
+          if (category.pipes.length > 0) {
+            category.pipes.sort((a, b) => {
+              if (a.name && b.name) {
+                const aName = a.name.toLowerCase();
+                const bName = b.name.toLowerCase();
+                if (aName > bName) {
+                  return 1;
+                }
+                if (aName < bName) {
+                  return -1;
+                }
+              }
+              return 0;
+            });
+          }
+        });
+        if (data !== null) {
```

Comment:
The other benefit of using the null default is you don't need this conditional either.   You can just do the commit no matter what the value is.